packages/apps/http-cache/values.yaml

```diff
@@ -1,4 +1,10 @@
 external: false
+
+haproxy:
+  replicas: 2
+nginx:
+  replicas: 3
```

Comment:
Why do you need three replicas? two replicas are enough for minimum fault tolerance.

Reply:
fixed, thank you